ExploreASL_Initialize.m

```diff
@@ -27,287 +21,249 @@
 % xASL_init_FileSystem             - dirty initialization of common filenames used throughout the pipeline
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % EXAMPLE for GUI: ExploreASL_Initialize
-% EXAMPLE for calling externally: ExploreASL_Initialize('//MyDisk/MyStudy/DataPar.m', true, 1, 1);
-% EXAMPLE for debugging/initialization only: [x] = ExploreASL_Initialize('',0);
+% EXAMPLE for calling externally: ExploreASL_Initialize('//MyDisk/MyStudy/DataPar.m');
+% EXAMPLE for debugging/initialization only: [x] = ExploreASL_Initialize('',0,0);
 % __________________________________
-% Copyright 2015-2020 ExploreASL
+% Copyright 2015-2021 ExploreASL
 
 
+    %% Admin
 
-%% Admin
+    % Define input parser
+    p = inputParsing(varargin{:});
```

Comment:
There's a comment in the header to check out ExploreASL_Master regarding the input arguments, since ExploreASL_Master & ExploreASL_Initialize share the same input arguments now. I can specify it in both scripts if you prefer that, but I actually prefer declaring them once, otherwise we always have to update both descriptions if there is a change.

Reply:
Yes that looked good